build_tools/release_notes/6.0.0_notes.txt

```diff
@@ -0,0 +1,101 @@
+* Plot2D instances are now of `Plotter2DWidget` type. Fixes #2586 by @rozyczko in https://github.com/SasView/sasview/pull/2587
+* fix dialog sizes for some calculators. #2437 by @rozyczko in https://github.com/SasView/sasview/pull/2581
+* Fix for getting directory name by @rozyczko in https://github.com/SasView/sasview/pull/2596
+* Remove Unused Dependency: h5py by @gdrosos in https://github.com/SasView/sasview/pull/2585
+* 2577 orientation viewer doesnt work from nigthly build on mac by @lucas-wilkins in https://github.com/SasView/sasview/pull/2600
+* Fixed unmatched method signatures in Box&Wedge Interactor child classes by @ehewins in https://github.com/SasView/sasview/pull/2589
+* Particle editor by @lucas-wilkins in https://github.com/SasView/sasview/pull/2520
+* Pass the Data1D/2D object, not its `data` attribute by @rozyczko in https://github.com/SasView/sasview/pull/2592
+* Bump reportlab from 3.6.6 to 3.6.13 in /build_tools by @dependabot in https://github.com/SasView/sasview/pull/2597
+* Added 3.11, removed 3.8 by @rozyczko in https://github.com/SasView/sasview/pull/2582
+* Fix doc build errors by @smk78 in https://github.com/SasView/sasview/pull/2607
+* Doc toctree fixes by @smk78 in https://github.com/SasView/sasview/pull/2609
+* 2603: Numeric coercion in preferences by @krzywon in https://github.com/SasView/sasview/pull/2605
+* What's new dialog by @lucas-wilkins in https://github.com/SasView/sasview/pull/2608
+* created submenu for slicers being part of #2604 by @astellhorn in https://github.com/SasView/sasview/pull/2610
+* Squish squashed by @lucas-wilkins in https://github.com/SasView/sasview/pull/2616
+* Update sas_gen.py by @timsnow in https://github.com/SasView/sasview/pull/2617
+* Killed Zombie Python Test by @lucas-wilkins in https://github.com/SasView/sasview/pull/2627
+* Remove model.png by @lucas-wilkins in https://github.com/SasView/sasview/pull/2629
+* Populate whats new with last version by @smk78 in https://github.com/SasView/sasview/pull/2625
+* Fix errors while running convertUI by @krzywon in https://github.com/SasView/sasview/pull/2623
+* 2618: Fix GPU and Optimizer Preferences by @krzywon in https://github.com/SasView/sasview/pull/2622
+* Moving OSX signing to nightly by @wpotrzebowski in https://github.com/SasView/sasview/pull/2631
+* Empty lines in data explorer by @rozyczko in https://github.com/SasView/sasview/pull/2643
+
+## New Contributors
+* @smalex-z made their first contribution in https://github.com/SasView/sasview/pull/2535
+* @ehewins made their first contribution in https://github.com/SasView/sasview/pull/2543
+* @lozanodorian made their first contribution in https://github.com/SasView/sasview/pull/2558
+* @gdrosos made their first contribution in https://github.com/SasView/sasview/pull/2585
+* @astellhorn made their first contribution in https://github.com/SasView/sasview/pull/2610
+
+**Full Changelog**: https://github.com/SasView/sasview/compare/nightly-build...v6.0.0-alpha
```

Comment:
Should this full changelog URL compare 5.0.6 to 6.0.0-alpha? Comparing to the nightly-build tag (which was tagged at some point during codecamp Oct 2022, from what I can see) doesn't seem right. 

I suspect there's also a stack of PRs that aren't listed in the What's Changed section, perhaps because they were merged prior to the `nightly-build` tag but after the branch point for `release_5.0.6`? (I've not looked in detail, but I think all the stuff I was doing in late 2022 wasn't destined for 5.0.6 but was held over for 6.0.0 and isn't listed there)

Reply:
Yes, the notes themself need to be worked on. This is more of the template. The main point of this is that release notes disappear every time someone pushes to the branch, so this is mostly to prevent empty records on this page: https://github.com/SasView/sasview/releases/tag/v6.0.0-alpha